fe/fe-core/src/main/java/org/apache/doris/analysis/ShowResourceGroupsStmt.java

```diff
@@ -0,0 +1,73 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.ResourceGroupMgr;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.UserException;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSetMetaData;
+
+import java.util.List;
+
+public class ShowResourceGroupsStmt extends ShowStmt {
+
+    private Expr whereClause;
+    private LimitElement limitElement;
+    private List<OrderByElement> orderByElements;
+
+    public ShowResourceGroupsStmt(Expr labelExpr, List<OrderByElement> orderByElements, LimitElement limitElement) {
+        this.whereClause = labelExpr;
+        this.orderByElements = orderByElements;
+        this.limitElement = limitElement;
+    }
+
+    @Override
+    public void analyze(Analyzer analyzer) throws UserException {
+        super.analyze(analyzer);
+    }
+
+    @Override
+    public String toSql() {
+        return "SHOW RESOURCE GROUPS";
+    }
+
+    @Override
+    public String toString() {
+        return toSql();
+    }
+
+    @Override
+    public ShowResultSetMetaData getMetaData() {
+        ShowResultSetMetaData.Builder builder = ShowResultSetMetaData.builder();
+        for (String title : ResourceGroupMgr.RESOURCE_GROUP_PROC_NODE_TITLE_NAMES) {
```

Comment:
We should be able to see more details in resource group, such as pending jobs, running jobs.
See comments in #14428

Reply:
may be implemented later